core/modules/cart/helpers/createMethodsData.ts

```diff
@@ -0,0 +1,44 @@
+import ShippingDetails from '@vue-storefront/core/modules/checkout/types/ShippingDetails'
+import { ShippingMethodsData } from '@vue-storefront/core/modules/cart/types/ShippingMethodsData'
+
+const getShipping = (shippingMethods: any[]) => {
+  const shipping = shippingMethods && Array.isArray(shippingMethods) ? shippingMethods.find(item => item.default && !item.offline) : null
+
+  if (!shipping && shippingMethods && shippingMethods.length > 0) {
+    return shippingMethods.find(item => !item.offline)
+  }
+
+  return shipping
+}
+
+const getPayments = (paymentMethods: any[]) => {
+  const payment = paymentMethods && Array.isArray(paymentMethods) ? paymentMethods.find(item => item.default) : null
+
+  if (!payment && paymentMethods && paymentMethods.length > 0) {
+    return paymentMethods[0]
+  }
+
+  return payment
+}
+
+const createMethodsData = (shippingDetails: ShippingDetails, shippingMethods: any[], paymentMethods: any[], defaultCountry: string): ShippingMethodsData => {
```

Comment:
name of method is not saying what exactly this is and where should we use it, would be nice to change it somehow.
Also it's a good practice to provide default values and take destructured object as params :)